src/chapters/booleans-and-conditionals/booleans.rst

```diff
@@ -0,0 +1,209 @@
+
+There are only two **boolean values**. They are ``true`` and ``false``. As always, JavaScript is case-sensitive, so ``True`` and ``False`` are not valid boolean values.
+
+.. admonition:: Example
+
+   .. sourcecode:: js
+
+      console.log(true);
+      console.log(typeof true);
+      console.log(typeof false);
+
+   **Output**
+
+   ::
+
+      true
+      boolean
+      boolean
+
+It is extremely important to realize that ``true`` and ``false`` are **not** strings. Surrounding the words "true" and "false" in quotes in JavaScript will result in string values rather than boolean values. 
+
+.. admonition:: Example
+
+   .. sourcecode:: js
+
+      console.log(typeof true);
+      console.log(typeof "true");
+
+   **Output**
+
+   ::
+
+      boolean
+      string
+
+Boolean Conversion
+------------------
+
+.. index:: ! Boolean(), type conversion
+
+As with the number and string data types, the boolean type has a type conversion function, ``Boolean()``. It works similarly to ``Number()`` and ``String()``, attempting to convert a non-boolean value to a boolean.
+
+.. admonition:: Try It!
+
+   Explore how ``Boolean()`` converts various non-boolean values.
+
+   .. sourcecode:: js
+
+      console.log(Boolean("true"));
+      console.log(Boolean("TRUE"));
+      console.log(Boolean(0));
+      console.log(Boolean(1));
+      console.log(Boolean(''));
+      console.log(Boolean('LaunchCode'));
+
+   `Run this program at repl.it <https://repl.it/@launchcode/Boolean-Type-Conversion>`_
+
+   Under which conditions does ``Boolean()`` convert a string to ``true``?
+
+   #. Only when the string is ``"true"``.
+   #. Whenever the string contains any non-whitespace character.
+   #. Whenever the string in non-empty.
+   #. Never. It converts all strings to ``false``.
+
+
+Boolean Expressions
+-------------------
+
+.. index::
+   single: boolean; expression
+
+.. index::
+   single: operator; equality
+
+.. index:: ! ==
+
+A **boolean expression** is an expression that evaluates to a boolean value. The equality operator, ``==``, compares two values and produces a boolean value related to whether the two values are equal to one another.
+
+.. admonition:: Example
+
+   .. sourcecode:: js
+
+      console.log(5 == 5);
+      console.log(5 == 6);
+
+   **Output**
+
+   ::
+
+      true
+      false
+
+In the first statement, the two operands are equal, so the expression evaluates to ``true``. In the second statement, 5 is not equal to 6, so we get ``false``.
+
+We can also use ``==`` to see that ``true`` and ``"true"`` are not equal.
+
+.. admonition:: Example
+
+   .. sourcecode:: js
+
+      console.log(true == "true");
+
+   **Output**
+
+   ::
+
+      false
+
+.. index::
+   single: operator; comparison
+
+The ``==`` operator is one of six common **comparison operators**.
+
+.. index:: ==, ! !=, ! <, ! >, ! <=, ! >=
+
+.. list-table:: Comparison Operators
```

Comment:
The table title "Comparison Operators" appears as light-grey on the webpage.  Could we darken the text to make it easier to see?  Alternatively, the table title seems redundant given the intro line, so the title could be removed without losing clarity.

Reply:
I added this styling issue to [Trello](https://trello.com/c/8pkRJaXt). The title is a mandatory field for the RST directive `list-table`. We could hide such titles universally, but it strikes me that in other cases they may be necessary and useful. I'm inclined to leave them as-is for now, but we can raise the issue at a broader scale later on, if need-be.